types/auth0/index.d.ts

```diff
@@ -509,13 +509,64 @@ export interface StatsParams {
   to: string;
 }
 
+export type Job = ImportUsersJob | ExportUsersJob | VerificationEmailJob;
+
+export type JobFormat = 'csv' | 'json';
+
+export type JobStatus = 'pending' | 'completed';
```

Comment:
I would assume there is another status for "running" or "in_progress", but I've never seen it and I can't find anything in the Auth0 docs to suggest it exists/what the value is

Reply:
I found "processing"and "failed" mentioned here https://community.auth0.com/t/management-api-we-need-better-docs/18910/2